lib/xboxrt/libc_extensions/string_ext_.c

```diff
@@ -0,0 +1,16 @@
+#include <string.h>
+#include <stdlib.h>
+
+char *strdup (const char *s)
```

Comment:
Why is this not just in functions/string/strdup.c instead?

Reply:
Imho it makes sense to keep it separate to make it clear that this is a downstream extension (upstream only want C functionality, not POSIX), and to make PDCLib easier to rebase by keeping the changes to non-xbox code small.

Reply:
`strdup` was only added in C2x is very recent and I assume pdclib will take some time to support it (if it's even planned).

So I think this is fine to keep it in our tree for now (as GNU/POSIX/... extension to earlier C standards, which are common versions we compile with).